File: Competive Programming/Ruby/HackerEarth/bubble_sort.rb
len = $stdin.gets.strip.to_i
arr = $stdin.gets.strip.split.map { |n| n.to_i }
swaps = 0


for i in 0...(len-1)
    for j in 0...(len-1)
        if arr[j] > arr[j+1]
            arr[j], arr[j+1] = arr[j+1], arr[j]
            swaps += 1
        end
    end
end


$stdout.puts swaps

File: Competive Programming/Python/HackerEarth/bubble-sort.py
arr_len = int(input())
arr = list(map(int, input().strip().split()))
swaps = 0


for i in range(arr_len-1):
    for j in range(arr_len-1):
        if arr[j] > arr[j+1]:
            arr[j], arr[j+1] = arr[j+1], arr[j]
            swaps += 1


print(swaps)
